integr8ly/delorean
# MGDAPI-5585 initial documentation

Jira: https://issues.redhat.com/browse/MGDAPI-5585

Verification steps:
- Follow the steps for single-az private link.
- No need to install RHOAM - just confirm that cluster can be accessed via bastion SSM/SSH/Browser
